deploy/nebula/templates/clusterdefinition.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterDefinition
+metadata:
+  name: nebula
+  labels:
+    {{- include "nebula.labels" . | nindent 4 }}
+spec:
+  connectionCredential:
+    username: root
+    password: "$(RANDOM_PASSWD)"
+    endpoint: "$(SVC_FQDN):$(SVC_PORT_tcp-nebula)"
+    host: "$(SVC_FQDN)"
+    port: "$(SVC_PORT_tcp-nebula)"
+    accesskey: ""
+    secretkey: ""
+  componentDefs:
+    - name: nebula-graphd
+      configSpecs:
+        - name: nebula-graphd-config
+          templateRef: nebula-graphd-config-template
+          volumeName: nebula-graphd
+      scriptSpecs:
+        - name: nebula-graphd-scripts
+          templateRef: nebula-graphd-scripts-template
+          volumeName: scripts
+          defaultMode: 493
+      workloadType: Stateful
+      characterType: nebula-graphd
+      service:
+        ports:
+          - name: thrift
+            port: 9669
+            protocol: TCP
+            targetPort: 9669
+          - name: http
+            port: 19669
+            protocol: TCP
+            targetPort: 19669
+          - name: http2
+            port: 19670
+            protocol: TCP
+            targetPort: 19670
+      podSpec:
+        securityContext:
+          fsGroup: 1001
+        containers:
+          - name: nebula-graphd
+            volumeMounts:
+              - mountPath: /usr/local/nebula/data
+                name: data
+              - mountPath: /usr/local/nebula/logs
+                name: logs
+              - mountPath: /usr/local/nebula/etc
+                name: nebula-graphd
+              - name: scripts
+                mountPath: /scripts
+            command:
+              - /bin/bash
+              - -c
+              - |
+                set -ex
+                /scripts/nebula-graphd-start.sh
+            imagePullPolicy: {{default .Values.nebula.graphd.image.pullPolicy "IfNotPresent"}}
+            ports:
+            - containerPort: 9669
+              name: thrift
+              protocol: TCP
+            - containerPort: 19669
+              name: http
+              protocol: TCP
+            - containerPort: 19670
+              name: http2
+              protocol: TCP
+            readinessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /status
+                port: 19669
```

Comment:
use port name instead

Reply:
check them all

Reply:
thanks. fixed. used port names in all `httpGet`.